rosdep/python.yaml

```diff
@@ -1648,6 +1648,13 @@ python-fcn-pip: &migrate_eol_2025_04_30_python3_fcn_pip
     pip:
       depends: [liblapack-dev]
       packages: [fcn]
+python-filfinder-pip:
```

Comment:
do you want to add the python2 or the python3 entry?
For python3 please use `python3-filfinder-pip`

Reply:
resolved